_notes/nipraxis_spring_22.md

```diff
@@ -42,7 +42,7 @@ Your teachers are themselves researchers in brain imaging, and contributors to b
 ## Prerequisites
 
 You should know some very basic programming in any language. In particular, we
-will expect you to know what variables are, and For Loops, and how to call
+will expect you to know what variables and [for loops](https://en.wikipedia.org/wiki/For_loop) are, and how to call
```

Comment:
The caps seemed to be to indicate that this needed some clarification for the readers who don't know what they are, so I figured a link would do the job as well and read more naturally.